presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleDistributedQueries.java

```diff
@@ -47,9 +47,13 @@ protected QueryRunner createQueryRunner()
             throws Exception
     {
         this.oracleServer = new TestingOracleServer();
-        return OracleQueryRunner.createOracleQueryRunner(
-                oracleServer,
-                TpchTable.getTables());
+        return createOracleQueryRunner(oracleServer, TpchTable.getTables());
+    }
+
+    protected QueryRunner createOracleQueryRunner(TestingOracleServer server, Iterable<TpchTable<?>> tables)
+            throws Exception
+    {
+        return OracleQueryRunner.createOracleQueryRunner(server, tables);
```

Comment:
Inline this back it is no longer needed .

Reply:
done